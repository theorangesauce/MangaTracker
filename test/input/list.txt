--List Test------
L
L
L
O
N
L
O
A
L
O
P
L
C
L
I
L
G
L
W
--Show empty series--
O
5
Y
--List (2)-----------
L
L
L
N
L
A
L
P
L
C
L
I
L
G
L
W
--Hide empty series--
O
5
N
X
--END LIST TEST------
