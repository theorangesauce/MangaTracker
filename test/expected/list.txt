Show Empty Series in Lists disabled:

  List All:
    Last Series
    Series 1
    Series 2
    Series 3
    Series 4

  List by Name:
    Last Series
    Series 1
    Series 2
    Series 3
    Series 4

  List by Author:
    Series 1
    Series 2
    Series 3
    Series 4
    Last Series

  List by Publisher:
    Series 1
    Series 4
    Series 2
    Series 3
    Last Series

  List Completed:
    Last Series
    Series 2

  List Incomplete:
    Series 1
    Series 3
    Series 4

  List Gaps:
    Series 3
    Series 4

  List Wishlist:
    Series 5


Show Empty Series in Lists enabled:

  List All:
    Last Series
    Series 1
    Series 2
    Series 3
    Series 4
    Series 5

  List by Name:
    Last Series
    Series 1
    Series 2
    Series 3
    Series 4
    Series 5

  List by Author:
    Series 1
    Series 2
    Series 3
    Series 5
    Series 4
    Last Series

  List by Publisher:
    Series 1
    Series 4
    Series 5
    Series 2
    Series 3
    Last Series

  List Completed:
    Last Series
    Series 2

  List Incomplete:
    Series 1
    Series 3
    Series 4
    Series 5

  List Gaps:
    Series 3
    Series 4

  List Wishlist:
    Series 5
